css/index.css

```diff
@@ -175,19 +175,109 @@ footer {
 }
 
 
+/* media queries */
+
+/* tablet*/
+
+
+@media(max-width: 768px){
+    .container{
+        width: 100%;
+        margin: 10px 200px 100px 10px;
+
```

Comment:
It would have been nice to see this as padding instead of margin. Margin slightly pushes elements off the side when looking at 768px view.